docs/_layouts/blog.html

```diff
@@ -1,26 +1,24 @@
 ---
 layout: default
 ---
-
-<div class="home">
-  
-  {{ content }}
-
-  <ul class="post-list">
+<section class="blog">
+    {{ content }}
+    <ul class="list">
     {% for post in site.posts %}
-      <li>
-        <span class="post-meta">{{ post.date | date: "%b %-d, %Y" }}</span>
-
-        <h2>
-          <a class="post-link" href="{{ post.url | relative_url }}">{{ post.title | escape }}</a>
-        </h2>
-        <span class="post-meta">
-          {{ post.content | strip_html | escape | truncatewords: 80 }}
-        </span>
-      </li>
+        <li>
+            <time class="time">
+                {% assign date_format = site.date_format | default: "%b %-d, %Y" %}
```

Comment:
The purpose of doing this is to make it easy to change the time format and not to make mistakes even if the settings are wrong.